samples/snippets/src/main/java/com/example/monitoring/CreateTimeSeries.java

```diff
@@ -0,0 +1,104 @@
+import com.google.monitoring.v3.CreateTimeSeriesRequest;
+import com.google.monitoring.v3.Point;
+import com.google.monitoring.v3.ProjectName;
+import com.google.monitoring.v3.TimeInterval;
+import com.google.monitoring.v3.TimeSeries;
+import com.google.monitoring.v3.TypedValue;
+import com.google.protobuf.util.Timestamps;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+// Sample to create time series
+public class CreateTimeSeries {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    createTimeSeries(projectId);
+  }
+
+  public static void createTimeSeries(String projectId) throws IOException {
+    // Instantiates a client
+    try (MetricServiceClient metricServiceClient = MetricServiceClient.create()) {
+      // Prepares an individual data point
+      TimeInterval interval =
+          TimeInterval.newBuilder()
+              .setEndTime(Timestamps.fromMillis(System.currentTimeMillis()))
+              .build();
+      TypedValue value = TypedValue.newBuilder().setDoubleValue(123.45).build();
+      Point point = Point.newBuilder().setInterval(interval).setValue(value).build();
+
+      List<Point> pointList = new ArrayList<>();
+      pointList.add(point);
+
+      // Prepares the metric descriptor
+      Map<String, String> metricLabels = new HashMap<>();
+      Metric metric =
+          Metric.newBuilder()
+              .setType("custom.googleapis.com/my_metric")
+              .putAllLabels(metricLabels)
+              .build();
+
+      // Prepares the monitored resource descriptor
+      Map<String, String> resourceLabels = new HashMap<>();
+      resourceLabels.put("instance_id", "1234567890123456789");
+      resourceLabels.put("zone", "us-central1-f");
+
+      MonitoredResource resource =
+          MonitoredResource.newBuilder()
+              .setType("gce_instance")
+              .putAllLabels(resourceLabels)
+              .build();
+
+      // Prepares the time series request
+      TimeSeries timeSeries =
+          TimeSeries.newBuilder()
+              .setMetric(metric)
+              .setResource(resource)
+              .addAllPoints(pointList)
+              .build();
+
+      List<TimeSeries> timeSeriesList = new ArrayList<>();
+      timeSeriesList.add(timeSeries);
+
+      CreateTimeSeriesRequest request =
+          CreateTimeSeriesRequest.newBuilder()
+              .setName(ProjectName.of(projectId).toString())
+              .addAllTimeSeries(timeSeriesList)
+              .build();
+
+      // Writes time series data
+      metricServiceClient.createTimeSeries(request);
+      System.out.println("Done writing time series value");
+    } catch (ApiException ex) {
+      System.out.print("\n time series could not be written." + ex.toString());
+    }
```

Comment:
\n as string constants get picked up by static analyzers.
If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

Reply:
Done